tools/clang/test/CodeGenSPIRV/meshshading.ext.no-explicit-exts-spv1.4.mesh.hlsl

```diff
@@ -0,0 +1,33 @@
+// RUN: %dxc -T ms_6_5 -E main -fspv-target-env=vulkan1.1spirv1.4 -fcgl %s -spirv | FileCheck %s
```

Comment:
nit: I'd be in favor of removing `-fcgl` so the optimizations (and the capability trimming pass are run)
This required a small fix in the other test (suggested below)

```suggestion
// RUN: %dxc -T ms_6_5 -E main -fspv-target-env=vulkan1.1spirv1.4 %s -spirv | FileCheck %s
```

Reply:
I added a 2nd commit